<launch>

  <node pkg="planning_utils" type="planner_node.py" name="planner_node" output="screen">
    <param name="static_map" type="string" value="static_map" />
    <param name="halton_points" type="int" value="500" />
    <param name="disc_radius" type="int" value="100" />
    <param name="source_topic" type="string" value="/initialpose" />
    <param name="target_topic" type="string" value="/move_base_simple/goal" />
    <param name="pub_topic" type="string" value="/planner_node/car_plan" />
  </node>
</launch>
